src/d3d9/d3d9_device.cpp

```diff
@@ -1478,8 +1478,14 @@ namespace dxvk {
         for (auto rt : m_state.renderTargets) {
           auto rtv = rt != nullptr ? rt->GetRenderTargetView(srgb) : nullptr;
 
-          if (unlikely(rtv != nullptr))
+          if (unlikely(rtv != nullptr)) {
             ClearImageView(fullClear, offset, extent, rtv, VK_IMAGE_ASPECT_COLOR_BIT, clearValueColor);
+
+            D3D9CommonTexture* dstTexture = rt->GetCommonTexture();
```

Comment:
Just do rt->GetCommonTexture()->IsAutomaticMip()

Reply:
Then I need to call rt->GetCommonTexture() twice, MarkTextureMipsDirty needs it as well.

Reply:
derp...